doc/book/service-manager.md

```diff
@@ -73,6 +82,21 @@ the configuration (`MyLogger`):
 $logger = $container->get('MyLogger');
 ```
 
+Notes:
+
+- The keys of the writers are not required, but they make the merge of the
+config files easier. The same remark can be made for the list of processors,
+filters, etc.
+
+- The key `priority` next to the key `name` should not be mingled with the
```

Comment:
This is already explained in the introduction: https://docs.zendframework.com/zend-log/intro/#using-built-in-priorities

Reply:
Here, this is an application of this definition. And the difference between the priority in the queue and the priority as a severity filter should be explained.

Reply:
@Daniel-KM 
See my comment above.